mkdocs.yml

```diff
@@ -1,7 +1,8 @@
-site_name: 'Remote Event Extension'
-repo_url: https://github.com/systopia/de.systopia.remoteevent
```

Comment:
Do we want to keep the `repo_url`?

Reply:
Yes, [repo_url](https://www.mkdocs.org/user-guide/configuration/#repo_url) isn't the same as [site_url](https://www.mkdocs.org/user-guide/configuration/#site_url).